### PR TITLE
Update issue template and readme

### PR DESCRIPTION
**Pull request by 5chdn**
- Ask about connected network in issue template
- Link the new Parity community on Riot in readme

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ _Before filing a new issue, please **provide the following information**._
 > - **Which operating system?**: Windows / MacOS / Linux
 > - **How installed?**: via installer / homebrew / binaries / from source
 > - **Are you fully synchronized?**: no / yes
+> - **Which network are you connected to?**: ethereum / ropsten / kovan / ...
```
Comment (jacogr) [inline .github/ISSUE_TEMPLATE.md]: s/ethereum/foundation/ ?

File: README.md [hunk 2]
```diff
@@ -31,7 +34,7 @@ Parity comes with a built-in wallet. To access [Parity Wallet](http://web3.site/
 
 By default, Parity will also run a JSONRPC server on `127.0.0.1:8545` and a websockets server on `127.0.0.1:8546`. This is fully configurable and supports a number of APIs.
 
-If you run into an issue while using parity, feel free to file one in this repository or hop on our [Gitter](https://gitter.im/paritytech/parity) or [Riot](https://riot.im/app/#/room/#parity-watercooler:matrix.org) chat room to ask a question. We are glad to help!
+If you run into an issue while using parity, feel free to file one in this repository or hop on our [Gitter](https://gitter.im/paritytech/parity) or [Riot](https://riot.im/app/#/group/+parity:matrix.parity.io) chat room to ask a question. We are glad to help!
```
Comment (jacogr) [inline README.md]: "If you run into an issue while using Parity"

instead?

Comment (5chdn) [reply README.md]: Uppercase Parity?

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (5chdn) [reply .github/ISSUE_TEMPLATE.md]: I think Ethereum is still a valid and well-understood reference to the primary public network. :)

Comment (jacogr) [reply .github/ISSUE_TEMPLATE.md]: 100%, just mentioned it since we use "foundation" as the main chain name in our software.

**Pull request merged**